### PR TITLE
Added prompt to select between eslint and jshint

### LINKED ISSUE #195: move to eslint
this should move to eslint (per the latest internal paypal usage)

### PR DESCRIPTION
**Pull request by aravindsrivats**
This PR resolves #195 by adding an option to choose between eslint and jshint.

### CODE REVIEW

File: app/templates/dependencies/eslint/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        ],
+        "semi": [
+            2,
+            "always"
+        ],
+        "no-console": 1
+    },
+    "env": {
+        "node": true,
+        "browser": true
```
Comment (grawk) [inline app/templates/dependencies/eslint/.eslintrc]: Is this common to set both node and browser to `true`?

File: test/app-build.js [hunk 2]
```diff
@@ -79,34 +87,67 @@ describe('kraken:app', function () {
...
-            build.on('close', function (code) {
+            build.on('close', function(code) {
                 assert.strictEqual(code, 0);
                 done(err);
             });
         });
     });
 
-    it('scaffolded application does not suffer from dll hell with dust-helpers', function (done) {
+    it('scaffolded application does not suffer from dll hell with dust-helpers', function(done) {
```
Comment (grawk) [inline test/app-build.js]: noticing we lost spaces between `function` keyword and parentheses for anon functions.

**Comment by grawk** [comment]
There may be some good changes in here (like addition of an `.eslintrc` file. But, I don't really think we want to up the complexity of the generator for choice of linting tool. ESLint is the recommended solution within the PayPal node.js community so I'm inclined to move straight to it and not offer an option.

**Comment by grawk** [comment]
If you'd like to alter this PR to simply move over to ESLint, then we can work with it.

File: test/app-build.js (hunk 2 above)
Comment (aravindsrivats) [inline test/app-build.js]: Yeah, bad.  My editor uses auto formatting with options that didn't match the source.  Will fix 'em up in the PR.

File: app/templates/dependencies/eslint/.eslintrc (hunk 1 above)
Comment (aravindsrivats) [inline app/templates/dependencies/eslint/.eslintrc]: I set both to true to enable lint on public/js files.  Maybe I'll remove it out of the .eslintrc and can be specified on the documentation.

**Comment by xjamundx** [comment]
Agreed eslint should be default

**Comment by grawk** [comment]
I wonder if the test failure is due to some incompatibility with v0.10 that these new changes introduce into the dependency tree?

**Comment by grawk** [comment]
subsumed by #205. released as `v2.2.2`

**Pull request closed without merging**